### PR TITLE
feat: add external control to tooltip

### PR DESCRIPTION
**Pull request by noahwaldner**
- add external control props to tooltip

CU-8694v8rz3

### CODE REVIEW

File: .changeset/loud-waves-fly.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/fondue-components": patch
```
Comment (SamuelAlev) [inline .changeset/loud-waves-fly.md:2]:
```suggestion
"@frontify/fondue-components": minor
```

File: packages/components/src/components/Tooltip/Tooltip.tsx
Comment (SamuelAlev) [inline packages/components/src/components/Tooltip/Tooltip.tsx:1]: Please add a test case

File: packages/components/src/components/Tooltip/Tooltip.stories.tsx [hunk 2]
```diff
@@ -50,3 +51,30 @@ export const LongDelay: Story = {
         enterDelay: 2000,
     },
 };
+
+export const ExternalControl: Story = {
```
Comment (SamuelAlev) [inline packages/components/src/components/Tooltip/Tooltip.stories.tsx:55]:
```suggestion
export const ControlledComponent: Story = {
```

File: packages/components/src/components/Tooltip/Tooltip.tsx
Comment (noahwaldner) [reply packages/components/src/components/Tooltip/Tooltip.tsx:1]: test added

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**